libminifi/test/libtest/integration/HTTPHandlers.cpp

```diff
@@ -453,18 +453,33 @@ void HeartbeatHandler::verifySupportedOperations(const rapidjson::Document& root
   REQUIRE(operations == std::set<std::string>(magic_enum::enum_names<minifi::c2::Operation>().begin(), magic_enum::enum_names<minifi::c2::Operation>().end()));
 }
 
-bool StoppingHeartbeatHandler::handlePost(CivetServer *, struct mg_connection *conn) {
-  verify(conn);
-  sendStopOperation(conn);
-  return true;
-}
-void StoppingHeartbeatHandler::sendStopOperation(struct mg_connection *conn) {
-  std::string resp = "{\"operation\" : \"heartbeat\", \"requested_operations\" : [{ \"operationid\" : 41, \"operation\" : \"stop\", \"operand\" : \"2438e3c8-015a-1000-79ca-83af40ec1991\"  }, "
-      "{ \"operationid\" : 42, \"operation\" : \"stop\", \"operand\" : \"FlowController\"  } ]}";
+void StoppingHeartbeatHandler::sendStartStopOperation(struct mg_connection *conn) {
+  std::lock_guard<std::mutex> lock(post_count_mutex_);
+  std::string requested_operation;
+  if (post_count_ == 0) {
+    requested_operation = R"({ "operationid" : 41, "operation" : "stop", "operand" : "2438e3c8-015a-1000-79ca-83af40ec1991" })";
+  } else if (post_count_ == 1) {
+    requested_operation = R"({ "operationid" : 42, "operation" : "stop", "operand" : "FlowController"  })";
+  } else if (post_count_ == 2) {
+    requested_operation = R"({ "operationid" : 43, "operation" : "start", "operand" : "2438e3c8-015a-1000-79ca-83af40ec1991" })";
```

Comment:
previously we sent both operations in a single request, what prompted the change?

Reply:
I think I just started testing them separately, and added them one by one, but I think the flow and component start or stop operations can be merged to make the test run faster, I updated in https://github.com/apache/nifi-minifi-cpp/pull/2057/commits/4ce50e373a42eba19a68e33a6716bcc17468a60d